app/components/versions-graph.js

```diff
@@ -55,7 +58,7 @@ export default Component.extend({
     ];
 
     // Update plotData with rows in the correct format for google visualization library
-    for (var [key, value] of downloadsPerVersion) {
+    for (var [key, value] of sortIncreasingBySemver(downloadsPerVersion)) {
       plotData.push([key, value, '#62865f', value]);
```

Comment:
Maybe in here we could change the trailing `.0` to `.x` before pushing into `plotData`?

Reply:
I've implemented this change in the latest pushed commit.